htdocs/helpFiles/Entering-Essays.html

```diff
@@ -0,0 +1,12 @@
+<h2 style="text-align: center">Entering Essays</h2>
+
+<p>
+	In an essay answer text box, you can type a long answer. After after you hit submit, it will be saved so that your
+	instructor can read it and assign a score at a later time.
+</p>
+<p>If your instructor makes any comments on your answer, those comments will appear on the exercise page.</p>
```

Comment:
I find "on the exercise page" a bit strange. I would call it the "problem page", or maybe just as the help before this does, "on this page". Maybe specify the comment will be at the top of the page.